aspects/aspects.go

```diff
@@ -153,9 +153,9 @@ func NewAspectBundle(account string, bundleName string, aspects map[string]inter
 	}
 
 	for name, v := range aspects {
-		accessPatterns, ok := v.([]map[string]string)
+		accessPatterns, ok := v.([]interface{})
 		if !ok {
-			return nil, fmt.Errorf("cannot define aspect %q: access patterns should be a list of maps", name)
+			return nil, fmt.Errorf("cannot define aspect %q: access patterns should be a list", name)
```

Comment:
list of maps was fine I think,  the error need to explain what is expected beyond what the code is checking